locales/en/messages.json

```
@@ -4807,6 +4807,14 @@
     "osdDescElementVtxChannel": {
         "message": "Current VTX channel and power"
     },
+    "osdTextElementVTXchannelVariantpower": {
```

To be camelCase need the P uppercase:
```suggestion
    "osdTextElementVTXchannelVariantPower": {
```